packages/react-filerobot-image-editor/src/actions/selectTool.js

```
@@ -1,12 +1,14 @@
 export const SELECT_TOOL = 'SELECT_TOOL';
 
-const selectTool = (state, payload) =>
-  state.toolId === payload.toolId
+const selectTool = (state, payload) => {
+  console.log('selectTool', {payload, state});
```

ESLint

[error] Replace `payload,·state` with `·payload,·state·`

(prettier/prettier)